deploy/kubernetes/images/prow-gke-release-staging-head/image.yaml

```diff
@@ -12,8 +12,8 @@ kind: ImageTagTransformer
 metadata:
   name: imagetag-csi-provisioner-prow-head
 imageTag:
-  name: gke.gcr.io/csi-provisioner
-  newName: quay.io/k8scsi/csi-provisioner
+  name: k8s.gcr.io/sig-storage/csi-provisioner
```

Comment:
Are there any alpha/dev image transformations that need to be updated?

Reply:
No. Alpha only overrides the driver image, not the sidecars. And dev just uses alpha image transformers.